go/oasis-test-runner/oasis/fixture.go

```diff
@@ -225,6 +225,7 @@ type RuntimeFixture struct { // nolint: maligned
 	Kind       registry.RuntimeKind `json:"kind"`
 	Entity     int                  `json:"entity"`
 	Keymanager int                  `json:"keymanager"`
+	Version    uint64               `json:"version"`
```

Comment:
Would be better if this was either a `version.Version` or a string which gets parsed into a `version.Version`.

Reply:
done.